### PR TITLE
modify help message for prerequisite

### PR DESCRIPTION
**Pull request by jimmyliao**
Signed-off-by: Jimmy <jimmyliao@jimmyliao.net>

** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

File: install/primehub-prerequisite [hunk 1]
```diff
@@ -32,17 +32,16 @@ error() {
...
-  $SELF check ingress          : check Ingress
-  $SELF check gpu              : check GPU
-  $SELF check storageclass     : check StorageClass
-
-  $SELF install linux_package  : check the Linux Package
   $SELF -h,--help              : show this message
 
+  $SELF install : check and install prerequisite if missing
+  $SELF check   : check only
+
```
Comment (LiuYuWei) [inline install/primehub-prerequisite:44]: @jimmyliao 
Please help to write it clearly
```
PURPOSE:
  Check the following pre-requisite demand:
  
  Command line:
  $SELF install <prerequisted-demand>: check and install prerequisite if missing
  $SELF check <prerequisted-demand>: check only
  
 prerequisted-demand Option:
  (1) linux_package 
  (2) container 
  (3) Kubernetes 
  (4) ingress 
  (5) gpu 
  (6) storageclass
```

Comment (jimmyliao) [reply install/primehub-prerequisite:44]: > @jimmyliao Please help to write it clearly
> 
> ```
> PURPOSE:
>   Check the following pre-requisite demand:
>   
>   Command line:
>   $SELF install <prerequisted-demand>: check and install prerequisite if missing
>   $SELF check <prerequisted-demand>: check only
>   
>  prerequisted-demand Option:
>   (1) linux_package 
>   (2) container 
>   (3) Kubernetes 
>   (4) ingress 
>   (5) gpu 
>   (6) storageclass
> ```

above Option: will confuse user to have more advance option to select

Comment (jimmyliao) [reply install/primehub-prerequisite:44]: The updated prerequisite is simply two part: check and install, the install will check all and bypass and continue next section if full-filled.

Comment (jimmyliao) [reply install/primehub-prerequisite:49]: please play around on your machine which not install any lib or partly installed. Thanks

Comment (LiuYuWei) [reply install/primehub-prerequisite:49]: @jimmyliao Got it. I just make some mistake here. I will merge the request.

**Pull request merged**